utils.py

```diff
@@ -205,6 +207,24 @@ def log_every(self, iterable, print_freq, header=None):
         print('{} Total time: {}'.format(header, total_time_str))
 
 
+def hex2rgb(hex: str):
+    return tuple(int(hex.lstrip('#')[i:i+2], 16) for i in (0, 2, 4))
+
+
+def create_colormap(labels: list) -> dict:
+    colormap = {0: (0, 0, 0)}
+    for i, label in enumerate(labels):
+        colormap[i+1] = hex2rgb(label['color'])
+    return colormap
```

Comment:
This is the new `create_colormap` which reflects the colormap of training dataset.
index 0 is the background.

Reply:
Is the attribute `label['color']` set by ABEJA Platform Annotation?

Reply:
Yes it is.